src/sagemaker/djl_inference/defaults.py

```diff
@@ -47,6 +47,9 @@
     "ml.g4dn",
     "ml.g5",
     "ml.p3",
+    "ml.p3dn",
```

Comment:
added by mistake?

Reply:
These got added from master when I pulled the latest changes while rebasing.